includes/admin/plugins.php

```diff
@@ -161,6 +163,12 @@ function perflab_render_plugins_ui() {
  * @return array{compatible_php: bool, compatible_wp: bool, can_install: bool, can_activate: bool} Availability.
  */
 function perflab_get_plugin_availability( array $plugin_data ): array {
+	if ( empty( $perflab_get_plugin_availability_plugins_checked ) ) {
```

Comment:
Is this missing a `global`? AFAIK otherwise the `empty()` check here will always be `false` otherwise.

```suggestion
	global $perflab_get_plugin_availability_plugins_checked;
	if ( empty( $perflab_get_plugin_availability_plugins_checked ) ) {
```